PWGHF/D2H/TableProducer/candidateCreatorCharmResoReduced.cxx

```diff
@@ -332,27 +348,121 @@ struct HfCandidateCreatorCharmResoReduced {
           rowCandidateResoTrack(pVecD[0], pVecD[1], pVecD[2],
                                 candV0Tr.px(), candV0Tr.py(), candV0Tr.pz(),
                                 invMassReso,
-                                invMassD);
+                                invMassD - invMassD0);
         } else {
-          rowCandidateReso(collision.globalIndex(),
-                           candD.globalIndex(),
-                           candV0Tr.globalIndex(),
-                           pVecD[0], pVecD[1], pVecD[2],
+          rowCandidateReso(pVecD[0], pVecD[1], pVecD[2],
                            pVecV0Tr[0], pVecV0Tr[1], pVecV0Tr[2],
                            invMassReso,
                            invMassD,
```

Comment:
now this is the delta mass, but I would put the mass instead (see comment above)